test/integration/targets/ansible-test/ansible_collections/ns/col/plugins/lookup/bad.py

```diff
@@ -0,0 +1,31 @@
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+name: bad
+short_description: Bad lookup
+description: A bad lookup.
+author:
+  - Ansible Core Team
+'''
+
+EXAMPLES = '''
+- debug:
+    msg: "{{ lookup('ns.col.bad') }}"
+'''
+
+RETURN = ''''''
```

Comment:
How about 
```suggestion
RETURN = ''
```

Reply:
I copied that from existing tests in the same collection. I've changed it to `''' # '''` as recommended here: https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block (8d8ecba47cfda2b94a958ef584d426a45e327db1).